apps/web/src/components/Logo/CurrencyLogo.tsx

```diff
@@ -40,6 +42,13 @@ export default function CurrencyLogo({
     return []
   }, [currency, uriLocations])
 
+  if (currency && currency.symbol && Object.keys(FIAT_CURRENCIES).includes(currency.symbol)) {
```

Comment:
don't rely on symbol. suggest move it out of currency logo

just render FiatIconLogo instead

Reply:
resolved. although instead of making a sperate fiatIcon component i added an isFiat prop and if this flag is true then it returns the curenncy loges from path based on symbol.